pac4j-saml/src/test/java/org/pac4j/saml/client/PostSAML2ClientTests.java

```diff
@@ -30,6 +32,16 @@ public void testCustomSpEntityIdForPostBinding() {
         final SAML2Client client = getClient();
         client.getConfiguration().setServiceProviderEntityId("http://localhost:8080/cb");
         client.getConfiguration().setUseNameQualifier(true);
+
+        final SAML2MetadataContactPerson person = new SAML2MetadataContactPerson();
```

Comment:
I'm not sure to understand if you test something here or if this information is now required (I though not).

Reply:
It's not required, but the block will make sure the changes are covered by the existing test when metadata is generated.